bigtable-hbase-2.x-parent/bigtable-hbase-2.x/pom.xml

```diff
@@ -149,6 +149,16 @@ limitations under the License.
       <version>${mockito.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.protobuf</groupId>
+      <artifactId>protobuf-java</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>grpc-google-cloud-bigtable-v2</artifactId>
+      <scope>test</scope>
```

Comment:
Both of these should be `runtime`:
They are build deps of a dep (bigtable-hbase-2x -> bigtable-hbase). However they werent listed here because until now they werent directly referenced in bigtable-hbase-2x. Now you are referencing them in a test.
Maven treats your declaration here as an override, which means that during runtime in production, the end user will get a missing class error. You have 2 options here:
1. change scope to runtime and add a comment (runtime is available in both test and runtime)
2. exclude the dep from dependency plugin and add comment

Reply:
hmm I think scope needs to be compile

Reply:
ok